python_modules/libraries/dagster-airbyte/dagster_airbyte/translator.py

```diff
@@ -140,4 +142,29 @@ class DagsterAirbyteTranslator:
 
     def get_asset_spec(self, props: AirbyteConnectionTableProps) -> AssetSpec:
         """Get the AssetSpec for a table synced by an Airbyte connection."""
-        raise NotImplementedError()
+        table_schema_props = props.json_schema.get("properties", {}) or props.json_schema.get(
+            "items", {}
+        ).get("properties", {})
+        column_schema = generate_table_schema(table_schema_props)
+
+        fully_qualified_table_name = (
+            f"{props.database}.{props.schema}.{props.stream_name}"
+            if props.database and props.schema
+            else None
+        )
```

Comment:
what's the context where this is actually none?

Given that this is entirely part of props maybe a property/method.